hbase-server/src/test/java/org/apache/hadoop/hbase/util/test/LoadTestDataGenerator.java

```diff
@@ -165,8 +165,7 @@ public Get beforeGet(long rowkeyBase, Get get) throws IOException {
   }
 
   /**
-   * Return the arguments passed to the generator as list of object
-   * @return
+   * @return Return the arguments passed to the generator as list of object
```

Comment:
Remove the `Return` word. It's redundant. Minor spelling mistake `... to the generator as a list of objects`.